packages/indexer-common/src/allocations/types.ts

```diff
@@ -175,3 +176,43 @@ export const parseGraphQLEpochs = (epoch: any): Epoch => ({
   totalIndexerRewards: epoch.totalIndexerRewards,
   totalDelegatorRewards: epoch.totalDelegatorRewards,
 })
+
+export interface NetworkEpochBlock {
+  network: string
+  epochNumber: number
+  startBlockNumber: number
+  startBlockHash: string
+}
+
+export const NetworkAliases: { [key: string]: string } = {
+  mainnet: 'eip155:1',
+  goerli: 'eip155:5',
+  gnosis: 'eip155:100',
+}
+
+export const aliasInverse = (eipFormat: string): string => {
```

Comment:
What does this function actually do? 

It returns an alias right?  What about `networkAlias()`? or `convertToAlias()`? or just `alias()`?

Reply:
Yep, specifically `NetworkAliases` converts the alias (human readable) to the network chain ids, and `aliasInverse` is backwards, converting ids to the alias. 
So I guess, maybe `NetworkAliases` -> `NetworkIDs` and `aliasInverse` -> `convertToAlias`?